src/components/Reports/ReportsView.tsx

```diff
@@ -57,11 +57,19 @@ export default function ReportsView(props: ReportsViewProps): JSX.Element {
   useEffect(() => {
     const refreshSearch = async () => {
       const reportsResults = await ReportService.getReports(selectedOrganization.id);
-      setResults(reportsResults.reports || []);
+      setResults(
+        (reportsResults.reports || []).map((report) => {
+          if (report.projectName) {
+            return report;
+          }
+          // Reports without a project name are for the organization
```

Comment:
can a project name be an empty string? wondering if we should key off the project id !== undefined

Reply:
The server doesn't allow it, but it looks like the project creation is not correctly validating that. I will fix that up in another PR.